mkt/feed/tests/test_views.py

```diff
@@ -924,23 +924,6 @@ def test_background_image(self, download_mock, crush_mock):
         eq_(crush_mock.call_args_list[0][1]['set_modified_on'][0], coll)
 
     @mock.patch('mkt.feed.fields.requests.get')
-    def test_background_image_crush(self, download_mock):
```

Comment:
@diox I deleted this test completely.  The other tests in this same file already mock out the pngcrush call so this just seemed redundant.